packages/appstore/src/components/modals/real-wallets-upgrade/components/content-with-link/content-with-link.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import classNames from 'classnames';
+import './content-with-link.scss';
+
+type TContentWithLinkProps = {
+    children: JSX.Element | Array<JSX.Element>;
+    is_mobile?: boolean;
+    is_right_forked?: boolean;
+    has_fork?: boolean;
+    fork_width?: number;
+    fork_margin?: number;
+};
+
+const ContentWithLink: React.FC<TContentWithLinkProps> = ({
+    children,
+    is_mobile,
+    is_right_forked,
+    has_fork,
+    fork_width = 15,
```

Comment:
can we just past the value with the units straight away instead?
```suggestion
    fork_width = 1.5rem,
```